docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -0,0 +1,321 @@
+- **uan-sshd**: Create and enable instances of `sshd` with systemd
+
+### Artifactory Assets
+
+UAN uploads artifacts to deploy to the UAN control-plane node in a new nexus repository:
+
+- uan-2.6.XX-third-party
+
+The following Nexus group repository is created and reference the aforementioned UAN Nexus raw repos.
+
+- uan-2.6-third-party
+
+This repository will contain the installer for K3s, Helm charts for HAProxy and MetalLB, etc.
+
+### Validation Tests 
+
+To validate the K3s cluster once deployed, see the [Validation Checks](#validation-checks) section of this document for details.
+
+## Configuring K3s, MetalLB, HAProxy, and SSHD for use with Podman
+
+Each of the sections below describe how the various comonents deployed to K3s and the UANs may be configured to enable users to SSH to rootless podman containers. As there is no one configuration to fit any one use case, read and understand each section to modify the configuration as needed. Once each section has been completed, see [Deploy K3s to the UAN](#deploy-k3s-to-the-uan).
+
+### MetalLB 
+
+Configure the start and end range for MetalLB `IPAddressPool` in `vars/uan_helm.yml`:
+```bash
+$ grep "^metallb_ipaddresspool" vars/uan_helm.yml
+metallb_ipaddresspool_range_start: "x.x.x.x"
+metallb_ipaddresspool_range_end: "x.x.x.x"
+```
+
+**Note**: In a future version of CSM, this range may be integrated into the System Layout Service (SLS) so the range will be automatically determined.
+
+**Important**: Before modifying `customer-access`, be sure to verify none of the IP Addresses in the new pool for UANs are being used by IMS or UAIs:
+```bash
+# kubectl get services -n ims | grep ims
+# kubectl get services -n user | grep uai
+# kubectl get services -n uai | grep 
+```
+
+It may be possible to reallocate the CSM MetalLB pool `customer-access` from CSM to make room for a subset of IPs to use with MetalLB on UANs. To shrink the `customer-access` pool in CSM, edit the configmap and pick a new CIDR block for for `customer-access`. In this example the CIDR block was `x.x.x.x/26`:
+```bash
+# kubectl edit -n metallb-system cm/metallb
+...
+data:
+  config: |
+    address-pools:
+    - addresses:
+      - x.x.x.x/27
+      name: customer-access
+      protocol: bgp
+...
+```
+This leaves a portion of IP Address unallocated that may then be used to set `metallb_ipaddresspool_range_start` and `metallb_ipaddresspool_end`. 
+
+**Important**: When calculating the range of IP Address now available from `customer-access`. Be sure to account for the Broadcast IP of the remaining `customer-access` pool.
+
+To complete migrating the IP Address range out of CSM, restart the MetalLB controller pod in CSM.
+```bash
+# kubectl delete pod -n metallb-system -l app.kubernetes.io/component=controller
+```
+
+### HAProxy Configuration
+Each SSH ingress into is backed by a K3s deployment of HAProxy. By default, a single instance of HAProxy is enabled in `vars/uan_helm.yml`:
```

Comment:
missing word between "into" and "is" in the first sentence. I think "UAN" should go there.